src/modules/roc_pipeline/receiver_session.cpp

```diff
@@ -158,7 +159,8 @@ ReceiverSession::ReceiverSession(const ReceiverSessionConfig& session_config,
 
         resampler_reader.reset(new (allocator) audio::ResamplerReader(
                                    *areader, *resampler_, sample_buffer_pool,
-                                   common_config.internal_frame_size),
+                                   common_config.internal_frame_length,
+                                   format->sample_rate, common_config.output_channels),
```

Comment:
One more problem: it should be `session_config.channels` (# of channels in input stream from network) instead of `common_config.output_channels` (# of channels in output stream to user).

Reply:
done.